Core/src/com/infiniteautomation/mango/spring/service/AuditEventService.java

```diff
@@ -69,7 +72,9 @@ public boolean hasReadPermission(PermissionHolder user, AuditEventInstanceVO vo)
 
     @EventListener
     protected void raiseAuditEvent(AuditEvent event) {
-        Common.backgroundProcessing.addWorkItem(new AuditEventWorkItem(event, this));
+        runAs.runAs(runAs.systemSuperadmin(), () -> {
```

Comment:
@MertCingoz I would want to be 100% sure of this change before you just bundle it with a different fix.  I haven't thought through the ramifications of doing this so unless you are confident in this change, remove it or do more testing.

Reply:
Removed